source/SIL.AppBuilder.Portal/src/routes/(authenticated)/projects/[id=idNumber]/+page.server.ts

```diff
@@ -156,6 +155,20 @@ export const actions = {
       return fail(403);
     const form = await superValidate(event.request, valibot(deleteAuthorSchema));
     if (!form.valid) return fail(400, { form, ok: false });
+    // TODO: Will this result in the desired behavior if a user has multiple roles?
```

Comment:
Possible solutions I have come up with:
1. Store a list of relevant roles in the user tasks entry. 
> Potentially cumbersome.
2. Prevent a user from having multiple roles. 
> Probably not the best for our use cases.
3. Don't add job to queue if the user is also the owner. 
> All the author-specific tasks are also owner tasks. This still wouldn't handle the edge case where an organization admin is also an author, but maybe this could be disallowed?

Neither one of these solutions is perfect, but a combination of these might be doable.